Tests/Schema.NET.Test/ValuesJsonConverterTest.cs

```diff
@@ -63,6 +63,30 @@ public void WriteJson_OneOrMany_GreaterThanOneCountWritesArray()
             Assert.Equal("{\"Property\":[\"A\",\"B\"]}", json);
         }
 
+        [Fact]
+        public void WriteJson_DateTime_ISO8601()
+        {
+            var value = new OneOrMany<DateTime>(new DateTime(2000, 1, 1, 12, 34, 56));
+            var json = this.SerializeObject(value);
+            Assert.Equal("{\"Property\":\"2000-01-01T12:34:56\"}", json);
+        }
+
+        [Fact]
+        public void WriteJson_DateTimeOffset_ISO8601()
+        {
+            var value = new OneOrMany<DateTimeOffset>(new DateTimeOffset(2000, 1, 1, 12, 34, 56, TimeSpan.FromHours(1)));
+            var json = this.SerializeObject(value);
+            Assert.Equal("{\"Property\":\"2000-01-01T12:34:56+01:00\"}", json);
+        }
+
+        [Fact]
+        public void WriteJson_TimeSpan_ISO8601()
+        {
+            var value = new OneOrMany<TimeSpan>(new TimeSpan(12, 34, 56));
+            var json = this.SerializeObject(value);
+            Assert.Equal("{\"Property\":\"12:34:56\"}", json);
```

Comment:
I don't think that `12:34:56` is ISO8601 format. As far as I know, this custom .NET format doesn't have a name. I think the test just needs renaming.

Reply:
I based it on the information on the [ISO 8601 Wikipedia page](https://en.wikipedia.org/wiki/ISO_8601#Times). Digging further, I found [a copy of the standard online](https://web.archive.org/web/20180702184405/http://www.webaugur.com/bibliotheca/standards/iso8601/att-8601v03.pdf) - on page 17, it refers to "HH:MM:SS" as the Time of Day format.

Effectively now Schema.NET supports both Durations and Time of Day under ISO 8601.

I will still rename it slightly so it refers to the "Time of Day" and not potentially misinterpreted as Duration.